src/Change_Problem.php

```diff
@@ -158,33 +160,21 @@ public static function showForProblem(Problem $problem)
         }
 
         if ($canedit) {
-            echo "<div class='firstbloc'>";
-
-            echo "<form name='changeproblem_form$rand' id='changeproblem_form$rand' method='post'
-                action='" . Toolbox::getItemTypeFormURL(__CLASS__) . "'>";
-
-            echo "<table class='tab_cadre_fixe'>";
-            echo "<tr class='tab_bg_2'><th colspan='3'>" . __('Add a change') . "</th></tr>";
-
-            echo "<tr class='tab_bg_2'><td>";
-            echo "<input type='hidden' name='problems_id' value='$ID'>";
-            Change::dropdown([
-                'used'        => $used,
-                'entity'      => $problem->getEntityID(),
-                'entity_sons' => $problem->isRecursive(),
-                'displaywith' => ['id'],
+            echo TemplateRenderer::getInstance()->render('components/form/link_existing_or_new.html.twig', [
+                'rand' => $rand,
+                'link_type' => __CLASS__,
+                'source_type' => Problem::class,
+                'source_id' => $ID,
+                'target_type' => Change::class,
```

Comment:
As `itemtype/items_id` are used everywhere in GLPI, it should be used here too.

```suggestion
                'link_itemtype' => __CLASS__,
                'source_itemtype' => Problem::class,
                'source_items_id' => $ID,
                'target_itemtype' => Change::class,
```